docs/source/inference.rst

```diff
@@ -0,0 +1,189 @@
+Inference Using Pretrained Embedding
+--------------------------------------
+
+Users can use DGL-KE to do inference tasks based on pretained embeddings (We recommand using DGL-KE to generate these embedding). Here we support two kinds of inference tasks:
+
+  * **Linkage score ranking** Given a list of (h, r, t) triplets, calculate the linkage score using the predefined score function for each triplet, sort the resulting scores and output the topk most confident triplets.
+  * **Embedding similarity ranking** Given a list of (e, e) enitity pairs or (r, r) relation pairs, calculate the similarity of for each pair, sort the resulting similarity score and output the topk most similar pairs.
+
+The ranking result will be automatically stored in the output file (result.tsv by default) using the tsv format.
+
+The Linkage Score Ranking
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The task of linkage score ranking is given a list of candidate (h, r, t) triplets, calculating the edge score of each triplet based on the trained model and the pretrained embeddings and then returning the topk most relevent triplets along with their scores. An example return value of top5 linkage score likes this::
```

Comment:
i'm confused if the command returns score of all input edges or returns top K ranked ones.

Reply:
By design it only returns top5.
Do we need an option for user to output all the scores?

Reply:
maybe --topk -1

Reply:
it seems the description is kind of redundant.